lib/fp.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.map = (list, ...args) => {
+  if (!list) throw new Error('array not provided');
+  if (typeof list === 'object')
```

Comment:
This if block currently doesn't do anything. There aren't braces defining its area of code and there is nothing to the right of it on the same line. Need to fix or remove.